CHANGES.md

```diff
@@ -5,6 +5,7 @@ Change Log
 
 ##### Fixes :wrench:
 * Fixed support of glTF-supplied tangent vectors. [#6302](https://github.com/AnalyticalGraphicsInc/cesium/pull/6302)
+* Added option `logDepthBuffer` to `Viewer`. With this option the globe is typically rendered in a single frustum using logarithmic depth. This helps to avoid artifacts on the connection of two frustums. [#5851](https://github.com/AnalyticalGraphicsInc/cesium/pull/5851)
```

Comment:
What is the right name with no abbreviations?  `logarithmicDepthBuffer` or just `logarithmicDepth`?

Also rewrite the description; this isn't just about the globe - it is everything.  It can also improve performance, not just visual quality.

Looking forward to those numbers.